src/utils/dom.ts

```diff
@@ -0,0 +1,15 @@
+export function isTouchDevice(): boolean {
+    if (!window) {
```

Comment:
This would error if window is not in the scope at all. Need something like that: `typeof window === 'undefined'`